### ISSUE TITLE
Missing dependency importlib_resources

### ISSUE THREAD

**Issue by unmonoqueteclea** [bigmlcom/sensenet#17]
As `importlib.resources` is only available in Python3.7+, you are doing this in `pretrained.py`

```
   try:
        import importlib.resources as pkg_resources
    except ImportError:
        import importlib_resources as pkg_resources
```

However, `setup.py` is not including the `importlib_resources` dependency, which is not part of Python's standard library

**Comment by charleslparker** [comment]
This was the thing that was nagging at my brain!

**Comment by charleslparker** [comment]
This should be fixed by fec5fdc.  I'm installing it just in case.

**Comment by unmonoqueteclea** [comment]
Fixed, thanks
